Final status:
Process still alive after running test suite
stdout and stderr:
DEBUG:root:[*] running method core_negotiate_tlv_encryption
DEBUG:root:[*] Negotiating TLV encryption
DEBUG:root:[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100df7685662460f26e274443befc628e1a3e9080997e85189455761ad866dac34d78a66fb05e214515f4d9fdb9033e5649ac50aaaa7513bf9ebe9e0b4d2cc62557ddedb7c7f8a40fd1a682dd42bf3e25ece064aafd945c10f3ee34fb4b9f317fda06300c961905a162f20f680b7cdaac187e204da7feebd8f3a1f815af0afb2cc8a5368259630013f606d3fd9df56a2d7449d2670323cf5cf263dcf18eac29ccd054826a937ae60e506049d5445062c4556f8e586f60a43fbcbe5017d88a039404146057b21005646367922feb397f5c83e66e957aff32d7f0a8d6254e6c50c6d1a43d2e4387fe0ffaa9ef53e60ec0f4d9ad3a775acd80003a1a1e95f7e465690b0203010001
DEBUG:root:[*] AES key: 0xa1165c06b112d0e1baa30e84befc3a2839ae0dba739dbb1491d8a5177016f9dc
DEBUG:root:[*] Encrypted AES key: 0x953e82cc25d68dacc61ebfbf30d024ea326279344c9804a3ffaee9bc4ee1f7e8a17232814e301ca1366b775e8686e55d5cb4be2a8d1dafbfe1793f9c3229f7af0864a54916921177a082181ddc5c0b1dc7d7982227b0625dd092fd45f64874a3777b7dc2bb76e8223fe803eb09f32bf22857b7ec16c6959f169334ac90725271e18205cc1dc11ff0ff583a6c314c3a92bf373daf77630db93d699d4cdd7db62b308c339ba9f8c2d4d04718734d9f92341f51935b542e4168527742456dc93d949626fae4a3e16a6a46e917626f2070c936012cc324c7ef4c5ee0cd806b2a0c8d2eaf03a0451ea248e3a61026955d04590c4add4f10a4ec16163c7468aa4614dd
DEBUG:root:[*] TLV encryption sorted
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_session_guid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_loadlib
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_sysinfo
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_uuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_interfaces
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_routes
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_separator
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-tvbgwyfi'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 1 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 2 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_ls
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo IFRXIOXQ']
DEBUG:root:[*] added process id: 4613, handle: 1
DEBUG:root:[*] added channel id: 3 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 4 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 4
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
DEBUG:root:[*] added process id: 4614, handle: 2
DEBUG:root:[*] added channel id: 5 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 6 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 6
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
DEBUG:root:[*] added process id: 4617, handle: 3
DEBUG:root:[*] added channel id: 7 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 8 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 8
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 9 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 9
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 10 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 10
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 11 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 11
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 12 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 12
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 13 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 13
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 14 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 14
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_file_move
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 15 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 15
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 16 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[-] method core_channel_close resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[-] method core_channel_close resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 16
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 17 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 17
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 18 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 18
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 19 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 19
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
